oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -427,8 +427,8 @@ public boolean createScopedRequired() {
   }
 
   /**
-   * If the credentials support scopes, creates a copy of the identity with the specified scopes;
-   * otherwise, returns the same instance.
+   * If the credentials support scopes, creates a copy of the identity with the specified scopes,
+   * invalidate access token (scoped); otherwise, returns the same instance.
```

Comment:
```suggestion
   * If the credentials support scopes, creates a copy of the identity with the specified scopes,
   * invalidates the existing scoped access token; otherwise, return the same instance.
```